src/main/index.tsx

```diff
@@ -162,15 +164,27 @@ const createWindow = async (): Promise<BrowserWindow> => {
     return window
 }
 
-// create main BrowserWindow when electron is ready
-app.on("ready", async () => {
-    win = await createWindow()
-    tray = createTray(app, win)
-    initializeAnalytics()
-    setupAnalyticsGlobals()
-    setupAnalyticsForApp(app)
-    autoUpdater.checkForUpdatesAndNotify()
-})
+if (!appInstanceLock) {
+    app.quit()
+} else {
+    app.on("second-instance", async () => {
+        // Someone tried to run a second instance, we should focus our window.
+        if (win) {
+            if (win.isMinimized()) win.restore()
+            win.focus()
```

Comment:
use win.show() here instead of .focus (same behavior as with tray icon double-click